frontend/appflowy_flutter/lib/plugins/document/presentation/plugins/plugins_export.dart

```diff
@@ -0,0 +1,21 @@
+export 'board/board_node_widget.dart';
```

Comment:
normally the barrel file has the same name of the folder that it's in. I'd rename this file to `plugins.dart`